main/templates/main/base_template.html

```diff
@@ -1,70 +1,100 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <title>{% block title %}roo.me{% endblock %}</title>
-    <!-- Bootstrap core CSS -->
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
 
-    <!-- Custom styles for this template -->
-    <link href="../../static/css/navbar.css" rel="stylesheet">
-  </head>
+<head>
+  <title>{% block title %}roo.me{% endblock %}</title>
+  <!-- Bootstrap core CSS -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-wEmeIV1mKuiNpC+IOBjI7aAzPcEZeedi5yW5f2yOq55WWLwNGmvvx4Um1vskeMj0" crossorigin="anonymous">
+
+  <!-- Custom styles for this template -->
+  <link href="../../static/css/navbar.css" rel="stylesheet">
+</head>
 
 <!--BODY-->
-  <body>
 
-    <!--NAV BAR-->
-    <nav class="navbar fixed-top navbar-expand-md navbar-light" style="background-color: #10d48e;">
+<body>
+
+  <!--NAV BAR-->
+  <nav class="navbar fixed-top navbar-expand-md navbar-light" style="background-color: #25d798;">
```

Comment:
You changed the color of the navbar? Is it the same as what Tamir did?

Reply:
I used Tamir's file in order to work on it. Everything he did on his PR you will see here too